CMakeLists.txt

```diff
@@ -81,9 +85,14 @@ if(CMAKE_COMPILER_IS_GNUCXX)
     endif()
 
     #set up compiler flags for GCC
+if (CMAKE_BUILD_TYPE MATCHES Debug)
+	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wno-attributes -O0") #support C++11 for std::, optimize
```

Comment:
check your indentation - there is a tab here (but the rest of the file uses spaces including your following line)